tests/config_test.py

```diff
@@ -52,16 +56,20 @@ def test_load_minigraph(self, get_cmd_module, setup_single_broacom_asic):
         result = runner.invoke(config.config.commands["load_minigraph"], ["-y"])
         print result.exit_code
         print result.output
+        traceback.print_tb(result.exc_info[2])
         assert result.exit_code == 0
         assert "\n".join([ l.rstrip() for l in result.output.split('\n')]) == load_minigraph_command_output
 
     def test_load_minigraph_with_disabled_telemetry(self, get_cmd_module, setup_single_broacom_asic):
         (config, show) = get_cmd_module
+        db = Db()
```

Comment:
here we create single db object and share the object for both config and show command so that the changes in the mockredis can preserve.

if we use multiple db connector, then everytime a new db connection is made, previous changes are not preserved.

Reply:
in case we do not need to share the db, then we do not need to pass the object explicitly. the click framework will automatically create one.